From 939a8058bd820109f6e5b2c3617811c83ec944cc Mon Sep 17 00:00:00 2001
From: Olivier Tilloy <olivier.tilloy@canonical.com>
Date: Mon, 12 Feb 2018 19:13:45 +0100
Subject: [PATCH] tdf#115659 Apparmor profile fixes for senddoc

Fix sending the current document by e-mail:
 - fix path to the installed senddoc executable
 - add missing exception for xdg-email
 - add missing include statement for sanitized_helper profile

Change-Id: Ie7c5b112bd6c415036490e9f0f81063ca3ced5a1
---

diff --git a/sysui/desktop/apparmor/program.senddoc b/sysui/desktop/apparmor/program.senddoc
index 0aa284d..4a40818 100644
--- a/sysui/desktop/apparmor/program.senddoc
+++ b/sysui/desktop/apparmor/program.senddoc
@@ -14,8 +14,9 @@
 
 #include <tunables/global>
 
-profile libreoffice-senddoc INSTDIR-/usr/lib{,32,64}/libreoffice/program/senddoc {
+profile libreoffice-senddoc INSTDIR-program/senddoc {
   #include <abstractions/base>
+  #include <abstractions/ubuntu-helpers>
 
   owner /tmp/lu**       rw,    #makes files like luRRRRR.tmp/lubRRRR.tmp where R is random
                                #Note, usually it's lub or luc, don't know why.
@@ -28,6 +29,7 @@
   /{usr/,}bin/grep      rmix,
   /{usr/,}bin/uname     rmix,
   /usr/bin/xdg-open     Cxr -> sanitized_helper,
+  /usr/bin/xdg-email    Cxr -> sanitized_helper,
   /dev/null             rw,
   INSTDIR-program/uri-encode rmpux,
   /usr/share/libreoffice/share/config/* r,
